docs/vm-installation.md

```diff
@@ -0,0 +1,78 @@
+# Deploying antrea-agent on VM
+
+Antrea-agent can be run on Linux/Windows VMs. Below are the steps needed to configure and run antrea-agent on VMs
+
+## Prerequisites on cluster
```

Comment:
To auto-generate pages in antrea.io, we need an empty line after title, bullet list, code/text/yaml blocks.

Reply:
"on Kubernetes cluster" to be clear.

Reply:
Done